CMakeLists.txt

```diff
@@ -59,3 +60,7 @@ endif ()
 if (BUILD_PDFNetPHP)
     add_subdirectory(${CMAKE_SOURCE_DIR}/PDFNetPHP)
 endif ()
+
+if (BUILD_PDFTronGo)
+    add_subdirectory(${CMAKE_SOURCE_DIR}/PDFTronGo)
```

Comment:
Is there a reason it's being filed under a different package name compared to the other language's wrappers?